parnamondal/Minimum number of jumps.java

```diff
@@ -1,43 +0,0 @@
-# Time Complexity - O(N)
-# Space Complexity - O(1)
-
-# JAVA SOLUTION 
```

Comment:
You've written codes in Python, why Java here?

Be language consistent.

Reply:
Sir , I intended to write the code in Python but some problem statments in GFG editor does not have the Python templates to write .
Also , you can check in this given link .

 [https://practice.geeksforgeeks.org/problems/minimum-number-of-jumps-1587115620/1](url)